.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
has this not been configured? https://github.com/701-T4/flatshare/blob/20993b15d41dc0ee98f4666fd023364804ceee1a/.prettierrc
just confused why this is empty

Reply:
ah yep I can add these settings in